### PR TITLE
Adm 375.3

### PR DESCRIPTION
**Pull request by tingwang1234**
## Summary

...

## Before

Before

## After

1. use interface to change e2e to sit
2. change get params for board(from '/board/Classic Jira' to '/board/classic-jira')
3. fix difference of unix timestamp in dateRangePicker
4. add e2e test for board when select classic jira

After

## Note

_Null_

### CODE REVIEW

File: frontend/src/clients/BoardClient.ts [hunk 1]
```diff
@@ -20,9 +20,8 @@ export class BoardClient extends HttpClient {
 
   getVerifyBoard = async (params: getVerifyBoardParams) => {
     try {
-      const result = await this.axiosInstance
-        .get(`/boards/${params.type}`, { params: { ...params } })
-        .then((res) => res)
+      const boardType = params.type === 'Classic Jira' ? 'classic-jira' : params.type.toLowerCase()
```
Comment (bhan-tw) [inline frontend/src/clients/BoardClient.ts:23]: If Type name in Params is different with boardType, it's better to create a mapping.

**Review by mengna-tw — APPROVED** [review]

File: frontend/src/clients/BoardClient.ts (hunk 1 above)
Comment (tingwang1234) [reply frontend/src/clients/BoardClient.ts:23]: right, I'll change this in next PR

File: backend/src/test/java/heartbeat/component/E2EUrlGeneratorTest.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Profile;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.net.URI;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest
```
Comment (bhan-tw) [inline backend/src/test/java/heartbeat/component/E2EUrlGeneratorTest.java:13]: Do not need @SpringBootTest @Profile("e2e") and ExtendWith(SpringExtension.class)

File: backend/src/test/java/heartbeat/component/E2EUrlGeneratorTest.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class E2EUrlGeneratorTest {
+
+	private final E2EUrlGenerator e2EUrlGenerator = new E2EUrlGenerator();
+
+	@Test
+	public void testGetUri() {
+		e2EUrlGenerator.setUrl("mockTest");
```
Comment (bhan-tw) [inline backend/src/test/java/heartbeat/component/E2EUrlGeneratorTest.java:15]: Could use ReflectionTestUtils to set the url in test, do not need add "@setter" in E2EUrlGenerator.
```
class E2EUrlGeneratorTest {

	@Test
	public void testGetUri() {
		String url = "http://test.com";
		E2EUrlGenerator e2EUrlGenerator = new E2EUrlGenerator();
		ReflectionTestUtils.setField(e2EUrlGenerator, "url", url);

		URI uri = e2EUrlGenerator.getUri("test-site");

		assertEquals(url, uri.toString());
	}

}
```

**Pull request merged**